be/src/olap/hll.cpp

```diff
@@ -164,7 +164,7 @@ size_t HyperLogLog::serialize(uint8_t* dst) const {
         break;
     }
     case HLL_DATA_EXPLICIT: {
-        DCHECK(_hash_set.size() < HLL_EXPLICIT_INT64_NUM)
+        DCHECK(_hash_set.size() <= HLL_EXPLICIT_INT64_NUM)
```

Comment:
DCECK will ignore in release build.  How this change can fix this bug?

Reply:
I use BUILD_TYPE=DEBUG to test.